AirCasting/SearchAndFollow/SearchLocation/SearchView.swift

```diff
@@ -18,6 +18,42 @@ struct SearchView: View {
         VStack(alignment: .leading) {
             title
             textField
+            parametersQuestion
+            HStack(spacing: 12) {
+                ForEach(ParameterType.allCases) { param in
+                    Button {
+                        withAnimation(.easeInOut(duration: 0.3)) {
+                            viewModel.onParameterTap(with: param)
+                        }
+                    } label: {
+                        Text(param.capitalizedName)
+                            .padding([.all], 5)
+                    }.buttonStyle(MultiSelectButtonStyle(isSelected: param.capitalizedName == viewModel.getParameter.capitalizedName))
+                        .padding(.bottom, 5)
+                }
+            }
+            sensorQuestion
+            if viewModel.shoudShowPMChoiceSheet {
```

Comment:
I think that should be way more flexible. Please design viewmodel so that if we want to add another sensor choice we won't need to make a single change to this view.